makefile.cargo

```diff
@@ -10,7 +10,7 @@ CONFIGURE_FLAGS := \
 
     CONFIGURE_FLAGS += \
         --with-default-fonts=/system/fonts \
-        --with-sysroot=$(ANDROID_TOOLCHAIN)/sysroot \
+        --with-sysroot=$(ANDROID_NDK)/platforms/android-18/arch-arm/ \
```

Comment:
Sure, I'd be OK with that! I assume you're proprosing that we would abort the compile if it's not set? I think that's workable, though then we might have to set it in https://github.com/servo/libfontconfig/blob/master/.travis.yml